webview/src/setup/components/experiments/Experiments.tsx

```diff
@@ -1,38 +1,39 @@
 import React from 'react'
-import { SectionCollapsed } from 'dvc/src/setup/webview/contract'
-import { showExperiments, showScmPanel } from './messages'
+import { useDispatch, useSelector } from 'react-redux'
 import { NoData } from './NoData'
 import { NeedsGitCommit } from './NeedsGitCommit'
-import { EmptyState } from '../../shared/components/emptyState/EmptyState'
-import { IconButton } from '../../shared/components/button/IconButton'
-import { Beaker } from '../../shared/components/icons'
-import { Button } from '../../shared/components/button/Button'
+import { showExperiments, showScmPanel } from '../messages'
+import { EmptyState } from '../../../shared/components/emptyState/EmptyState'
+import { IconButton } from '../../../shared/components/button/IconButton'
+import { Beaker } from '../../../shared/components/icons'
+import { Button } from '../../../shared/components/button/Button'
+import { updateSectionCollapsed } from '../../state/webviewSlice'
+import { SetupState } from '../../store'
 
 type ExperimentsProps = {
   isDvcSetup: boolean
-  hasData: boolean | undefined
-  setSectionCollapsed: (sectionCollapsed: SectionCollapsed) => void
-  needsGitCommit: boolean
 }
 
-export const Experiments: React.FC<ExperimentsProps> = ({
-  isDvcSetup,
-  hasData,
-  setSectionCollapsed,
-  needsGitCommit
-}) => {
+export const Experiments: React.FC<ExperimentsProps> = ({ isDvcSetup }) => {
+  const dispatch = useDispatch()
+  const { needsGitCommit, hasData } = useSelector(
+    (state: SetupState) => state.experiments
+  )
+
   if (!isDvcSetup) {
     return (
       <EmptyState isFullScreen={false}>
         <h1>DVC is not setup</h1>
         <p>DVC needs to be setup before you can access experiments.</p>
         <Button
           onClick={() =>
-            setSectionCollapsed({
-              dvc: false,
-              experiments: true,
-              studio: true
-            })
+            dispatch(
+              updateSectionCollapsed({
+                dvc: false,
+                experiments: true,
+                studio: true
+              })
```

Comment:
Out of scope, but in `Dvc.tsx` we have:

```
  [SetupSection.DVC]: true,
  [SetupSection.EXPERIMENTS]: false,
  [SetupSection.STUDIO]: false
```
We probably should keep using section keys.

Reply:
Good catch! I'll update that